authentication/views.py

```diff
@@ -0,0 +1,11 @@
+from django.shortcuts import render
+# Create your views here.
+
+
+def home(request):
```

Comment:
Can we make both of these class based views?

Reply:
Yes, Today i'm working on views and forms. I will covert these function views into class based views.